andela/bad_ass-ah-frontend
# #164489897 Follow and unfollow users

**What does this PR do?**

- This PR will allow a given user to follow or to unfollow others user

**Description of Task to be completed?**

-  User should be able to follow others
- User should be able to unfollow others
- User should be able to get all their followers
- User should be able to get all their following

**How should this be manually tested?**

- After clonning this repository, make sure the server is running or run it with the command `npm run dev`

- The register a user

- Make sure that more than `one` users are registered and post articles

- From the home page click on an article to read it

- To follow a user click on the button `follow`

- To unfollow a user click on the button `unfollow`

- To see all followers, go on the view profile page and click on `followers`

- To see all following, go on the view profile page and click on `following`

**What are the relevant pivotal tracker stories?**

[#164489897](https://www.pivotaltracker.com/story/show/164489897)

**Screenshots (if appropriate)**
![image](https://user-images.githubusercontent.com/46047244/59796247-556ac880-92dd-11e9-82c3-d5d661141032.png)

![image](https://user-images.githubusercontent.com/46047244/59796261-5e5b9a00-92dd-11e9-97c8-2140bcd3770e.png)

![image](https://user-images.githubusercontent.com/46047244/59796280-6fa4a680-92dd-11e9-82a6-25f8b11790a7.png)

![image](https://user-images.githubusercontent.com/46047244/59796294-74695a80-92dd-11e9-8126-9b528af1260e.png)

![image](https://user-images.githubusercontent.com/46047244/59796308-792e0e80-92dd-11e9-98a6-a9f136457c72.png)
